<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="admins-MacBook-Pro-2.local" failures="1" tests="3" name="Test" time="439.641" errors="0" timestamp="10 Apr 2018 15:06:17 GMT">
  <testcase classname="Test.Test2" name="SecondTask" time="195.92">
    <failure type="java.lang.StringIndexOutOfBoundsException" message="begin 0, end -4, length 0">
      <![CDATA[java.lang.StringIndexOutOfBoundsException: begin 0, end -4, length 0
	at java.base/java.lang.String.checkBoundsBeginEnd(String.java:3107)
	at java.base/java.lang.String.substring(String.java:1873)
	at Test.Test2.SecondTask(Test2.java:57)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:281)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:121)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- SecondTask -->
  <testcase classname="Test.Test3" name="thirdTest" time="152.367"/>
  <testcase classname="Test.Test1" name="FirstTask" time="85.001"/>
</testsuite> <!-- Test -->
